src/main/java/net/teumteum/auth/controller/OAuthLoginController.java

```diff
@@ -0,0 +1,26 @@
+package net.teumteum.auth.controller;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import net.teumteum.auth.domain.response.TokenResponse;
+import net.teumteum.auth.service.AuthService;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+@Slf4j
+@RequestMapping
+@RestController
+@RequiredArgsConstructor
+public class OAuthLoginController {
+
+    private final AuthService authService;
+
+    @PostMapping("/login/callback/{provider}")
```

Comment:
```suggestion
    @PostMapping("/logins/callbacks/{provider}")
```

이렇게 수정하는건 어떨까요? - REST API 컨벤션

Reply:
오 감사합니다!